### DISCUSSION TITLE
Where is unused-files-webpack-plugin used?

### DISCUSSION THREAD

**Discussion by SergioTx** [single-spa/create-single-spa#331] [Q&A] [answered]
I have seen it as a project dependency, but it is used nowhere in the project. I have checked other dependencies and still nothing.

This comes because I have an issue with that plugin that reports all the files that only contain TS interfaces as unused.

Is there a way to disable it in certain files? Or to disable it completely?

**Comment by internettrans** [comment, marked as answer]
unused-files-webpack-plugin doesn't work with webpack@5, so it was removed from webpack-config-single-spa in https://github.com/single-spa/create-single-spa/commit/efb694b196f626bf467213a81f48ca818d34dbb7#diff-790fd7329e3a3719755d15a22f897ed7e01866bb7b3407ea56f35f2584ca1fd3L97

It should no longer be a dependency added to newly generated projects, but seems it is. I'll create a github issue for this and put together a fix.

**Reply by internettrans** [reply]
Fix released in webpack-config-single-spa@5.1.1. See https://github.com/single-spa/create-single-spa/blob/main/packages/webpack-config-single-spa/CHANGELOG.md#511
